src/components/Card.js

```diff
@@ -1,19 +1,21 @@
 import React from 'react';
 
-import DownloadIcon from '@material-ui/icons/CloudDownload';
-import { ThemeProvider } from 'styled-components/macro';
+// import DownloadIcon from '@material-ui/icons/CloudDownload';
+// import { ThemeProvider } from 'styled-components/macro';
 import { makeStyles } from '@material-ui/core/styles';
 import Paper from '@material-ui/core/Paper';
 import Typography from '@material-ui/core/Typography';
 import Box from '@material-ui/core/Box';
 
 import { TextLoader, ChartLoader } from './Loader';
-import { theme } from './theme';
-import { ButtonStyle } from './Button';
+// import { theme } from './theme';
+// import { ButtonStyle } from './Button';
 import Chart from './Chart';
 
 import PropTypes from 'prop-types';
 
+const HEADER = 'Data is more powerful when shared';
```

Comment:
I see that you're no longer using this as a prop. This might be a case for a constants directory, at the same level as `/helpers` or `/components`. Something like `/constants/appHeader.js` so that it becomes easier for other developers to find and change.

Reply:
You should also remove the `header` proptype at the bottom of this component.